packages/core/upload/package.json

```diff
@@ -40,16 +40,13 @@
     "clean": "run -T rimraf dist",
     "lint": "run -T eslint .",
     "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
-    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
```

Comment:
Why remove these?

Reply:
We dont have the concept of EE/CE tests in the FE. legacy code